Hook up PreprocessAll flow to EdgeManager

mcr229 opened the pull request:
Summary: Enable Preprocess All to be used through the edgeprogrammanager

Differential Revision: D74629455

cccclai reviewed (approved):
Looks good to me! Will be great to have a test

mcr229 committed:
Hook up PreprocessAll flow to EdgeManager (#10842)

Summary:

Enable Preprocess All to be used through the edgeprogrammanager

Reviewed By: cccclai

Differential Revision: D74629455
